HISTORY.md

```diff
@@ -1,5 +1,8 @@
 # cloudpathlib Changelog
 
+## v0.8.1 (UNRELEASED)
```

Comment:
Let's bump to `0.9.0` since this is a compatible API change. Can you also update the version in `setup.py`?